### ISSUE TITLE
Kernal not starting after update 

### ISSUE THREAD

**Issue by gmmkmtgk** [spyder-ide/spyder#16697]
I just updated my spyder but the kernel is not starting after the update. 
Tried various commands like 
pip install ipykernal --upgrade 
pip install -U pyzmq
spyder --reset 
But not working even after the restart of PC two times

**Comment by gmmkmtgk** [comment]
Attached screenshot 
![Uploading image.png…]()

**Comment by ccordoba12** [comment]
Hey @gmmkmtgk, thanks for reporting. The image you tried to post didn't upload correctly.

Please upload it again.

**Comment by gmmkmtgk** [comment]
![Screenshot_2021-11-01-20-19-09-94](https://user-images.githubusercontent.com/51493190/139690884-4741dffd-2ce6-4c68-ba3f-b20b2a098bde.jpg)

**Comment by ccordoba12** [comment]
Ok, please open a terminal and run there

    python -m spyder_kernels.console

and post here the content printed in the console after that.

**Comment by gmmkmtgk** [comment]
did not work sir still kernal is stuck on loading 
![Screenshot from 2021-11-05 00-09-35](https://user-images.githubusercontent.com/51493190/140400561-397641b3-0510-4397-8fdf-df8e2b7f1872.png)
![Uploading Screenshot from 2021-11-05 00-13-46.png…]()

**Comment by ccordoba12** [comment]
What's the version of the `ipykernel` that got installed along with Spyder?

**Comment by gmmkmtgk** [comment]
I downloaded Spyder one year back at that time version of Spyder was 3. Then I upgraded it to 5. So I don't know wether the ipykernel has also been upgraded to 6.5.0 along with that

**Comment by ccordoba12** [comment]
Ok, please run in your terminal

    pip list

and post the result here to see what packages you have installed.

**Comment by gmmkmtgk** [comment]
Package                       Version
----------------------------- --------------------
absl-py                       0.13.0
aiofiles                      0.4.0
aiohttp                       3.7.4.post0
aioredis                      1.3.1
aiosmtplib                    1.1.4
alabaster                     0.7.12
amqp                          2.5.2
appdirs                       1.4.4
apturl                        0.5.2
arctic                        1.79.4
argon2-cffi                   20.1.0
arrow                         1.2.1
astroid                       2.6.6
astunparse                    1.6.3
async-generator               1.10
async-timeout                 3.0.1
atomicwrites                  1.4.0
attrs                         19.3.0
Automat                       20.2.0
autopep8                      1.5.7
Babel                         2.8.0
backcall                      0.1.0
bcrypt                        3.1.7
beautifulsoup4                4.10.0
billiard                      3.6.3.0
binaryornot                   0.4.4
black                         21.10b0
bleach                        3.3.0
blinker                       1.4
Brlapi                        0.7.0
cachetools                    4.2.4
celery                        4.4.2
celery-pool-asyncio           0.2.0
celery-redbeat                1.0.0
certifi                       2021.10.8
chardet                       3.0.4
charset-normalizer            2.0.7
click                         8.0.3
cloudpickle                   1.3.0
colorama                      0.4.4
command-not-found             0.3
conda                         4.3.16
constantly                    15.1.0
cookiecutter                  1.7.3
cryptography                  2.8
cupshelpers                   1.0
cycler                        0.10.0
databases                     0.2.6
DateTime                      4.3
dbus-python                   1.2.16
debugpy                       1.5.1
decorator                     4.4.2
defer                         1.0.6
defusedxml                    0.6.0
diff-match-patch              20200713
distlib                       0.3.1
distro                        1.4.0
distro-info                   0.23ubuntu1
dnspython                     1.16.0
docopt                        0.6.2
docutils                      0.16
duplicity                     0.8.12.0
email-validator               1.1.2
entrypoints                   0.3
enum-compat                   0.0.3
enum34                        1.1.10
eventlet                      0.25.2
fakeredis                     1.4.5
fastapi                       0.63.0
fastapi-mail                  0.3.5.0
fasteners                     0.14.1
filelock                      3.0.12
flake8                        3.9.2
Flask                         1.1.2
flatbuffers                   1.12
flower                        0.9.4
fsspec                        0.7.2
future                        0.18.2
fyers-api                     1.0.9
gast                          0.4.0
gevent                        20.5.0
google                        3.0.0
google-auth-oauthlib          0.4.4
google-pasta                  0.2.0
greenlet                      0.4.15
grpcio                        1.34.1
h11                           0.9.0
h5py                          3.1.0
hashids                       1.3.1
hiredis                       1.0.1
html5lib                      1.0.1
httpcore                      0.12.3
httplib2                      0.14.0
httptools                     0.1.1
httpx                         0.17.0
humanize                      0.5.1
idna                          2.8
imagesize                     1.2.0
importlib-metadata            1.7.0
incremental                   17.5.0
inflection                    0.5.1
install                       1.3.4
instamojo-wrapper             1.2.0
intervaltree                  3.1.0
ipykernel                     6.4.2
ipython                       7.29.0
ipython-genutils              0.2.0
ipywidgets                    7.6.3
isort                         4.3.4
itsdangerous                  1.1.0
jedi                          0.18.0
Jinja2                        2.11.3
jinja2-time                   0.2.0
joblib                        1.0.1
jsonpickle                    1.4.1
jsonschema                    3.2.0
jupyter                       1.0.0
jupyter-client                6.1.5
jupyter-console               6.4.0
jupyter-core                  4.6.3
jupyterlab-pygments           0.1.2
jupyterlab-widgets            1.0.0
keras-nightly                 2.5.0.dev2021032900
Keras-Preprocessing           1.1.2
keyring                       18.0.1
kiteconnect                   3.9.2
kiwisolver                    1.2.0
kombu                         4.6.8
language-selector             0.1
launchpadlib                  1.10.13
lazr.restfulclient            0.14.2
lazr.uri                      1.0.3
lazy-object-proxy             1.4.3
linecache2                    1.0.0
livereload                    2.6.3
lockfile                      0.12.2
logilab-common                1.4.3
louis                         3.12.0
lunr                          0.5.8
lxml                          4.6.3
lz4                           3.0.2
macaroonbakery                1.3.1
Mako                          1.1.0
Markdown                      3.3.4
MarkupSafe                    1.1.0
matplotlib                    3.3.1
matplotlib-inline             0.1.3
mccabe                        0.6.1
mistune                       0.8.4
mkdocs                        1.1.2
mkdocs-material               7.1.4
mkdocs-material-extensions    1.0.1
mock                          3.0.5
mockextras                    1.0.2
monotonic                     1.5
more-itertools                4.2.0
motor                         2.1.0
multidict                     5.1.0
mypy-extensions               0.4.3
nbclient                      0.5.3
nbconvert                     6.1.0
nbformat                      5.1.3
nest-asyncio                  1.5.1
netifaces                     0.10.4
nltk                          3.6.2
notebook                      6.4.0
nsepy                         0.8
numpy                         1.19.5
numpydoc                      0.7.0
oauthlib                      3.1.0
olefile                       0.46
opt-einsum                    3.3.0
packaging                     20.9
pandas                        1.0.3
pandas-ta                     0.2.45b0
pandocfilters                 1.4.3
paramiko                      2.6.0
parso                         0.8.2
passlib                       1.7.2
pathspec                      0.9.0
pbr                           5.4.5
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        7.0.0
pip                           21.3.1
platformdirs                  2.4.0
plotly                        4.6.0
pluggy                        1.0.0
portpicker                    1.2.0
poyo                          0.5.0
prometheus-client             0.11.0
prompt-toolkit                3.0.21
protobuf                      3.6.1
psutil                        5.5.1
ptyprocess                    0.6.0
pyasn1                        0.4.8
pyasn1-modules                0.2.8
pycairo                       1.16.2
pycodestyle                   2.7.0
pycosat                       0.6.3
pycparser                     2.20
pycups                        1.9.73
pydantic                      1.7.1
pydocstyle                    6.1.1
pyflakes                      2.3.1
Pygments                      2.6.1
PyGObject                     3.36.0
PyHamcrest                    2.0.2
PyJWT                         1.7.1
pylint                        2.9.6
pyls-spyder                   0.4.0
pymacaroons                   0.13.0
pymdown-extensions            8.1.1
pymongo                       3.10.1
PyNaCl                        1.3.0
pyOpenSSL                     19.1.0
pyparsing                     2.4.7
PyQt5                         5.12.3
PyQt5-sip                     12.9.0
PyQtWebEngine                 5.12.1
pyRFC3339                     1.1
pyrsistent                    0.18.0
pyshorteners                  1.0.1
pysvn                         1.9.9
python-apt                    2.0.0+ubuntu0.20.4.6
python-crontab                2.4.0
python-dateutil               2.8.1
python-debian                 0.1.36ubuntu1
python-digitalocean           1.15.0
python-dotenv                 0.13.0
python-lsp-black              1.0.0
python-lsp-jsonrpc            1.0.0
python-lsp-server             1.2.4
python-multipart              0.0.5
python-slugify                5.0.2
python-telegram-bot           12.4.2
pytrends                      4.7.3
pytz                          2019.3
pyxdg                         0.26
PyYAML                        5.3.1
pyzmq                         22.3.0
QDarkStyle                    3.0.2
qstylizer                     0.2.1
QtAwesome                     1.0.3
qtconsole                     5.1.1
QtPy                          1.9.0
redis                         3.4.1
regex                         2021.4.4
reportlab                     3.5.34
requests                      2.26.0
requests-unixsocket           0.2.0
retrying                      1.3.3
rfc3986                       1.4.0
roman                         2.0.0
rope                          0.10.5
rope-py3k                     0.9.4.post1
rsa                           4.7.2
Rtree                         0.9.7
ruamel.yaml                   0.17.17
ruamel.yaml.clib              0.2.6
scikit-learn                  0.24.2
scipy                         1.5.2
SecretStorage                 2.3.1
Send2Trash                    1.7.1
service-identity              18.1.0
setuptools                    54.1.1
simplegeneric                 0.8.1
simplejson                    3.16.0
sip                           4.19.21
six                           1.12.0
sklearn                       0.0
sniffio                       1.2.0
snowballstemmer               2.0.0
sortedcontainers              2.3.0
soupsieve                     2.2.1
Sphinx                        2.4.4
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        1.0.3
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.4
spyder                        5.1.5
spyder-kernels                2.1.3
SQLAlchemy                    1.3.15
starlette                     0.13.6
stocktrends                   0.1.4
systemd-python                234
ta                            0.5.17
tenacity                      6.2.0
tensorboard                   2.5.0
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.0
tensorflow                    2.5.0
tensorflow-estimator          2.5.0
tensorflow-gpu                2.5.0
termcolor                     1.1.0
terminado                     0.10.1
testpath                      0.5.0
testresources                 2.0.0
text-unidecode                1.3
textdistance                  4.2.2
threadpoolctl                 2.1.0
three-merge                   0.1.1
tinycss2                      1.1.0
toml                          0.10.2
tomli                         1.2.2
tornado                       4.5.3
tqdm                          4.60.0
traceback2                    1.4.0
traitlets                     5.0.5
truedata-ws                   3.0.2
Twisted                       20.3.0
typing-extensions             3.10.0.2
tzlocal                       2.0.0
ubuntu-advantage-tools        27.3
ubuntu-drivers-common         0.0.0
ufw                           0.36
ujson                         4.2.0
unattended-upgrades           0.1
unittest2                     1.1.0
upstox                        2.0.0
urllib3                       1.24.3
usb-creator                   0.3.7
uvicorn                       0.11.3
uvloop                        0.14.0
vboxapi                       1.0
vine                          1.3.0
virtualenv                    20.4.2
wadllib                       1.3.3
watchdog                      2.1.6
wcwidth                       0.1.8
webencodings                  0.5.1
websocket-client              0.58.0
websockets                    8.1
Werkzeug                      1.0.1
wheel                         0.36.2
widgetsnbextension            3.5.1
wrapt                         1.12.1
wsproto                       0.15.0
wurlitzer                     3.0.2
xkit                          0.0.0
yapf                          0.31.0
yarl                          1.6.3
zipp                          3.1.0
zope.interface                5.1.0

**Comment by ccordoba12** [comment]
Ok, please close Spyder, run in a terminal

    pip install jupyter_client==6.1.12

and try again.

**Comment by gmmkmtgk** [comment]
thanks a ton 
kernal started 
![Screenshot from 2021-11-08 00-12-32](https://user-images.githubusercontent.com/51493190/140657692-66a015c4-65ac-4bf0-8f58-923bc09e553c.png)

**Comment by ccordoba12** [comment]
Great! I'm really glad Spyder is working fine for you now.
